dashboard/test/controllers/musiclab_controller_test.rb

```diff
@@ -0,0 +1,8 @@
+require 'test_helper'
+
+class MusiclabControllerTest < ActionController::TestCase
+  test 'When DCDO flag is set to `true` channel ids from constant returns true ' do
```

Comment:
Is this a situation where writing an actual controller test doesn't work because of the issues we have with creating projects in tests?

Reply:
Yes. I believe we'd have to stub the db results similar to https://github.com/code-dot-org/code-dot-org/blob/50cb403a799fd3332020054bc74d9bd7d517e8ae/dashboard/test/lib/projects_list_test.rb#L502-L512.